.github/workflows/npm-publish.yml

```diff
@@ -22,3 +22,7 @@ jobs:
       run: npm publish --access public
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  deploy:
```

Comment:
Given that this package has more than one thing that it (for lack of better term) "distributes" — an `npm` package and, now, a web site that it deploys — I wonder if we might be more descriptive with the `deploy` workflow naming? e.g. `webDeploy` or something, to prevent it from getting confused with the package `publish`?

This might be stupid. And maybe we want to match `deploy` naming in other repos. Just...thinking.

Reply:
Just my 2p... I like standardisation, but only where it makes sense. In all the backed ish repos we have used the name `deploy`. In an ideal world we would here, but its not the end of the world if gets called something else that makes sense to this repository.

Reply:
Ignore me. I'm talking rubbish :-) Your comment was against another file. My comment specifically related to the `deploy.yml` workflow.

Reply:
I have changed it to `deploy-website`. I chose that over `webDeploy` because the other job in this workflow is `build-and-publish-package`, so I thought it would be more consistent.